cookbook/Chapter 2 - Selecting data & finding the most common complaint type.ipynb

```diff
@@ -36,7 +36,7 @@
    "outputs": [],
    "source": [
     "# because of mixed types we specify dtype to prevent any errors\n",
-    "complaints = pd.read_csv('../data/311-service-requests.csv', dtype='unicode')"
```

Comment:
Jupyterlite wants the entire tree under one directory, so I moved data under cookbook.